test/puppetlabs/trapperkeeper/services/status/status_service_test.clj

```diff
@@ -406,17 +406,28 @@
 (deftest status-status-test
   (testing "trapperkeeper-status registers its own status callback"
     (with-status-service
-     app []
-     (let [resp (http-client/get "http://localhost:8180/status/v1/services")]
-       (is (= 200 (:status resp)))
-       (is (= #{:status-service} (ks/keyset (parse-response resp true)))))
-     (let [resp (http-client/get "http://localhost:8180/status/v1/services/status-service?level=debug")]
-       (is (= 200 (:status resp)))
-       (let [body (parse-response resp true)]
-         (is (= {:detail_level "debug"
-                 :service_name "status-service"
-                 :service_status_version 1
-                 :service_version status-core/status-service-version
-                 :state "running"}
-                (dissoc body :status)))
-         (is (map? (get-in body [:status :experimental :jvm-metrics]))))))))
+      app []
```

Comment:
I don't care about this too much but it seems like you are using 2-space indent, and that is probably going to cause whitespace flip-flopping when other people come in behind you?